src/pages/iou/request/step/IOURequestStepDistanceRate.tsx

```diff
@@ -115,9 +132,15 @@ const IOURequestStepDistanceRateWithOnyx = withOnyx<IOURequestStepDistanceRatePr
     policy: {
         key: ({report}) => `${ONYXKEYS.COLLECTION.POLICY}${report ? report.policyID : '-1'}`,
     },
+    policyCategories: {
+        key: ({report}) => `${ONYXKEYS.COLLECTION.POLICY_CATEGORIES}${report ? report.policyID : '0'}`,
+    },
+    policyTags: {
+        key: ({report}) => `${ONYXKEYS.COLLECTION.POLICY_TAGS}${report ? report.policyID : '0'}`,
+    },
```

Comment:
Shoot, after thinking about this more I'm realizing that we would have to create a mapping between all reportIDs and their policy categories. That would be inefficient because we probably don't need that data for most reports.

We could do something wacky like call a function to set up a subscription in the action file, but then I we might run into a situation where the action is triggered needing the policyCategories before they are ready. I feel like eventually we will need a way to do something like this, but for now the team seems to think that's it's not a significant problem so I'm going to ignore it as well unfortunately. Sorry for the hassle. Do let me know if you have any other ideas though.